Automattic/jetpack
# Update "Widgets" and "Menus" links back to classic wp-admin destination

Update **"Appearance > Widgets"** and **"Appearance > Menus"** masterbar links back to classic wp-admin destination for Atomic and Simple sites.

After this update is applied:
<img src="https://user-images.githubusercontent.com/140841/136994250-cf9e987d-9b9f-401f-b8a5-012573bb441e.png" width="400" />

Fixes [#55388](https://github.com/Automattic/wp-calypso/issues/55388)

### Changes proposed in this Pull Request:
* Remove code that changes the **"Appearance > Widgets"** and **"Appearance > Menus"** links to point to `customize.php`.

### Jetpack product discussion
See issue [#55388](https://github.com/Automattic/wp-calypso/issues/55388)

### Does this pull request change what data or activity we track or use?
No.

### Testing instructions:
To test first verify you see the old links on your local install (control), then verify the links change after applying this update.

#### Test for simple site:

(Confirm control)
1. Apply the master branch to your WPCOM Sandbox.
2. (Using a sandboxed simple site.) Verify that "Appearance > Widgets" links to "wp-admin/customize.php?[...]widgets[...]"
3. (Using a sandboxed simple site.) Verify that "Appearance > Menus" links to "wp-admin/customize.php?[...]nav-menus[...]"

(Verify update)
1. Apply the patch D68295-code to your WPCOM Sandbox.
2. (Using the same sandboxed simple site.) Verify that "Appearance > Widgets" now links to "wp-admin/widets.php"
3. (Using the same sandboxed simple site.) Verify that "Appearance > Menus" now links to "wp-admin/nav-menus.php"

####  Test for Atomic site:

(Confirm control)
1. Checkout the Jetpack master branch.
2. Docker up and tunnel.
2. Install wpcomsh in your local Jetpack install.
3. Enable SSO on your local Jetpack install using Report Card.
4. Verify that "Appearance > Widgets" links to "wp-admin/customize.php?[...]widgets[...]"
5. Verify that "Appearance > Menus" links to "wp-admin/customize.php?[...]nav-menus[...]"

(Verify update)
1. Checkout this branch. `git checkout origin update/widgets-and-nav-menus-link`
2. Verify that "Appearance > Widgets" links to "wp-admin/widets.php"
3. Verify that "Appearance > Menus" now links to "wp-admin/nav-menus.php"
